### PR TITLE
Initial version of Kubernetes deployment variant

### PR DESCRIPTION
**Pull request by lulf**
Initial version of deployment script that works on public cloud providers. In the initial version, all nodes must be accessible on all ports on the public internet.

A second iteration of this work will make it possible to use LoadBalancer and Ingress for the appropriate resources.

For #58

### CODE REVIEW

**Review by dejanb — APPROVED** [review]

**Review by ctron — APPROVED** [review]

**Comment by ctron** [comment]
I will merge this once the PR #72 is merged.

**Pull request merged**
